shopping-cart/app/shoppingcart/checkout/address/CheckoutAddressPageController.java

```diff
@@ -107,71 +105,44 @@ protected boolean formHasErrors(final Form<CheckoutShippingAddressFormData> ship
         return sphere().execute(CartUpdateCommand.of(cart, updateActions));
     }
 
-    protected CompletionStage<Result> handleSuccessfulSetAddress(final UserContext userContext) {
-        final Call call = reverseRouter().showCheckoutShippingForm(userContext.locale().toLanguageTag());
+    protected CompletionStage<Result> handleSuccessfulSetAddress(final Cart cart) {
```

Comment:
👍 if you did it for consistency. I want to keep the same structure for the methods, even though I might not be using those parameters in my implementation. Because the point is to minimize the amount of overridden logic for others and for that I have to provide the typical information you would need.